lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.scss

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-threat-indicator-legend {
+  $max-grid-columns: 6;
+
+  align-items: center;
+  display: inline-grid;
+  gap: var(--nx-spacing-2x);
+  grid-template-columns: repeat($max-grid-columns, auto);
+
+  .nx-threat-indicator-legend-header {
+    grid-column: span $max-grid-columns;
+    margin: 0;
+  }
+
+  svg.nx-threat-indicator {
+    margin-left: 0;
+  }
+
+  &.nx-threat-indicator-legend--vertical {
+    grid-template-columns: none;
+    justify-items: flex-start;
```

Comment:
I'm not seeing a difference from this property

Reply:
> I'm not seeing a difference from this property

Removed unnecessary properties. 7865898a485adc6fbd4c6459d66b0c326c22f8c2